ui/lib/pki/addon/templates/keys/index.hbs

```diff
@@ -8,7 +8,7 @@
   }}
   @isEngine={{true}}
 />
-{{#if this.model.hasConfig}}
+{{#if (or this.model.hasConfig this.model.keyModels)}}
```

Comment:
There's an intermediate state after CSR's are generated where keys exist, but no issuers - we want to list those keys in that case